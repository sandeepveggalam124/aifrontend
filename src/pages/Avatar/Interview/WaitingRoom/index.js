import React from 'react'
import Timer from '../Timer'
import GoLink from '../../../../components/home/GoLink'

const WaitingRoom = () => {
  const interviewStartTime = new Date();
  interviewStartTime.setMinutes(interviewStartTime.getMinutes() + 1);
  return (
    <div className="row custom-div text-white">
    <div className='col col-lg-1 border-end d-flex flex-column align-items-center'>
        <GoLink/>
    </div>
    <div className='col col-lg-10 d-flex flex-column justify-content-around align-items-center'>
      <Timer expiryTimestamp={interviewStartTime} displaySize={100} showCountdown={true} onExpireNavigate="/interview/question" />

    </div>
    <div className='col col-lg-1 bg-last-image'></div>


    </div>
  )
}

export default WaitingRoom