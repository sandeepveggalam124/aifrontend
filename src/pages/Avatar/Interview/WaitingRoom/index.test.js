import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import WaitingRoom from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <WaitingRoom/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})