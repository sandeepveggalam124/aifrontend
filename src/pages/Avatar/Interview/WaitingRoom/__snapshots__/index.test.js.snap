// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="row custom-div text-white"
  >
    <div
      class="col col-lg-1 border-end d-flex flex-column align-items-center"
    >
      <div
        class="row text-white"
      >
        <div
          class="col col-lg-12 d-flex align-items-center flex-column "
        >
          <div
            class="py-5"
          >
            <img
              alt="Bootstrap"
              src="/assests/images/profile.svg"
            />
          </div>
          <div
            class="rotate-text p-5"
          >
            <a
              class="welcome-text highlighted-link"
              href="link_to_gostandout"
            >
              GoStandOut
            </a>
          </div>
          <div
            class="rotate-text p-5"
          >
            <a
              class="welcome-text "
              href="link_to_golearn"
            >
              GoLearn
            </a>
          </div>
          <div
            class="rotate-text p-5"
          >
            <a
              class="welcome-text "
              href="link_to_gototest"
            >
              GoToTest
            </a>
          </div>
          <div
            class="rotate-text p-5"
          >
            <a
              class="welcome-text "
              href="link_to_gosolvecode"
            >
              GoSolveCode
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="col col-lg-10 d-flex flex-column justify-content-around align-items-center"
    >
      <div
        class="d-flex flex-column justify-content-center align-items-center mt-2"
      >
        <div
          class="d-flex justify-content-center align-items-center"
        >
          <div
            style="font-size: 100px;"
          >
            <span>
              0
            </span>
            :
            <span>
              1
            </span>
            :
            <span>
              0
            </span>
          </div>
        </div>
        <div>
          <p>
            1 minutes left to start the interview
          </p>
        </div>
      </div>
    </div>
    <div
      class="col col-lg-1 bg-last-image"
    />
  </div>
</div>
`;
